### PR TITLE
AVRO-2826: Add .asf.yaml file to configure github notifications

### PR DESCRIPTION
**Pull request by iemejia**
ML thread for more details
https://lists.apache.org/thread.html/rada5d1b3dbee3469940d89d5c43042d008dcec026967c099fba04ade%40%3Cdev.avro.apache.org%3E

R: @RyanSkraba @Fokko

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - dotnet
+    - java
+    - .net
+    - perl
+    - php
+    - python
+    - ruby
+
+  enabled_merge_buttons:
+    squash: false
```
Comment (Fokko) [inline .asf.yaml:40]: I always use squash and merge, what do other use?

Comment (RyanSkraba) [reply .asf.yaml:40]: Well spotted!  I largely prefer squash-and-merge in 99% of the cases, and rarely (but occasionally) rebase.

Comment (iemejia) [reply .asf.yaml:40]: Ah good point, I don't know why I assumed that squash and merge created an additional merge commit but it does not seem to be the case
https://help.github.com/en/github/administering-a-repository/about-merge-methods-on-github

I will update this to let the two options that do not create the additional commit, well spotted @Fokko

**Comment by iemejia** [comment]
PR updated and rebased, I am waiting for the confirmation of the creation of the issues@ mailing list before merge (I added you @Fokko as a co-moderator even if it will be mostly unmoderated).

**Comment by iemejia** [comment]
testing

**Comment by iemejia** [comment]
testing message delivery to issues@

**Comment by iemejia** [comment]
test message 2

**Pull request merged**